Lib/queue.py

```diff
@@ -234,6 +234,13 @@ def _put(self, item):
 
     def _get(self):
         return heappop(self.queue)
+    
+    def __repr__(self):
+        max_length_to_display = 20
```

Comment:
There is a discussion on https://bugs.python.org/issue36049 on whether or not to have a short repr.

If it's decided to have a short repr, I don't think this need to be set as a variable, just write `if len(self.queue) > 20:`.